RecoTracker/TkSeedingLayers/src/SeedingLayerSetsBuilder.cc

```diff
@@ -185,7 +186,9 @@ SeedingLayerSetsBuilder::SeedingLayerSetsBuilder(const edm::ParameterSet & cfg,
 {
   std::vector<std::string> namesPset = cfg.getParameter<std::vector<std::string> >("layerList");
   std::vector<std::vector<std::string> > layerNamesInSets = this->layerNamesInSets(namesPset);
-
+  isFastSim = cfg.getParameter<bool>("isFastSim");
+  if(isFastSim)
+    fastSimrecHitsToken_ = iC.consumes<FastTrackerRecHitCollection>(edm::InputTag("fastTrackerRecHits"));
```

Comment:
Please change the `isFastSim` configuration parameter to a parameter to the constructor (with default value of `false`).

Reply:
Sorry, do you mean that I should not modify the value of isFastSim using a configuration parameter? Then how will I make sure that fastSimrecHitsToken_ is set only while running FastSim ?

Reply:
You can pass it as a parameter to the constructor.